aquarius/app/util.py

```diff
@@ -33,6 +34,15 @@ def make_paginate_response(query_list_result, search_model):
     return result
 
 
+def get_bool_env_value(envvar_name, default_value=0):
+    assert default_value in (0, 1), "bad default value, must be either 0 or 1"
+    try:
+        value = bool(int(os.getenv(envvar_name, default_value)))
```

Comment:
I think you can return it directly, instead of assigning it (in both the try and except blocks)